bin/build-image

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-image="nasa/harmony-gdal-adapter"
+image="ghcr.io/nasa/harmony-gdal-adapter"
```

Comment:
Third time is the charm. This now _definitely_ allows for testing of an updated image with a local Harmony instance.

Reply:
Actually we now can just pull image from ghcr.io.

Reply:
Absolutely, but if we want to make changes during development and then run those changes before merging or releasing them, the `bin/build-image` script needs to use the same full name as the release does to update the Docker image being used by Harmony locally.